src/main/java/core/basesyntax/model/shopstrategy/ShopTransactionsTypes.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model.shopstrategy;
+
+public enum ShopTransactionsTypes {
+    BALANCE("b"),
+    PURCHASE("p"),
+    RETURN("r"),
+    SUPPLY("s");
+    
+    private String value;
+    
+    ShopTransactionsTypes(String value) {
+        this.value = value;
+    }
+    
+    public static ShopTransactionsTypes getAction(String value) {
+        return java.util.Arrays.stream(ShopTransactionsTypes.values())
```

Comment:
the better remove java.util and import it above

Reply:
Did